Dockerfile

```diff
@@ -1014,6 +1014,107 @@ FROM scratch AS noble
 COPY --from=noble-build /tmp/google-cloud-ops-agent.tgz /google-cloud-ops-agent-ubuntu-noble.tgz
 COPY --from=noble-build /google-cloud-ops-agent*.deb /
 
+# ======================================
+# Build Ops Agent for ubuntu-oracular
+# ======================================
+
+FROM ubuntu:oracular AS oracular-build-base
+ARG OPENJDK_MAJOR_VERSION
+
+RUN set -x; apt-get update && \
+		DEBIAN_FRONTEND=noninteractive apt-get -y install git systemd \
+		autoconf libtool libcurl4-openssl-dev libltdl-dev libssl-dev libyajl-dev \
+		build-essential cmake bison flex file systemd-dev debhelper libsystemd-dev \
```

Comment:
Just a comment to make note that `debhelper` and `system-dev` had to be added here to make the build work.